<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartedDiskTab</class>
 <widget class="QWidget" name="PartedDiskTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Partitioning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PartitionDiskButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start GParted</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DiskPartitionSelect">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="DiskSelect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="FormatCheckBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Format Parition</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Disk</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Parititon</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FormatPartitionSelect">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="MyDeviceIsEFICheckBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string> It's a EFI system</string>
   </property>
  </widget>
  <widget class="QComboBox" name="MyDeviceIsEFIComboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MyDeviceIsEFILabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EFI System Parition</string>
   </property>
  </widget>
  <widget class="QLabel" name="FileSystemTypeLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>File System</string>
   </property>
  </widget>
  <widget class="QLabel" name="UseSwapLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Swap Parition</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="UseSwapCheckBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Use Swap</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SwapSelect">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>UseSwapCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>SwapSelect</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseSwapCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>UseSwapLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
